src/acp/util/setVcAccess.test.ts

```diff
@@ -198,4 +198,98 @@ describe("setVcAccess", () => {
       ]).graphs.default
     );
   });
+
+  it("should set the given access mode on the member access control for recursive VC access", () => {
+    const resource = mockAccessControlledResource();
+
+    expect(resource.internal_acp.acr.graphs).toStrictEqual({ default: {} });
+    const updatedResource = setVcAccess(
+      resource,
+      { read: true },
+      { inherit: true }
+    );
+    expect(updatedResource.internal_acp.acr.graphs.default).toStrictEqual(
+      createDatasetFromSubjects([
+        [
+          DEFAULT_ACCESS_CONTROL_RESOURCE_URL,
+          [
+            [ACP.accessControl, [TEST_URL.defaultAccessControl]],
+            [
+              ACP.memberAccessControl,
+              [
+                // Setting recursive access creates a member access control.
+                TEST_URL.defaultMemberAccessControl,
+              ],
+            ],
+          ],
+        ],
+        [
+          TEST_URL.defaultAccessControl,
+          [[ACP.apply, [TEST_URL.defaultAccessControlVcPolicy]]],
+        ],
+        [
+          TEST_URL.defaultMemberAccessControl,
+          // The member access control should enforce the VC access policy.
+          [[ACP.apply, [TEST_URL.defaultAccessControlVcPolicy]]],
+        ],
+        [
+          TEST_URL.defaultAccessControlVcPolicy,
+          [
+            [rdf.type, [ACP.Policy]],
+            [ACP.anyOf, [TEST_URL.defaultAccessControlVcMatcher]],
+            [ACP.allow, [ACL.Read]],
+          ],
+        ],
+        [
+          TEST_URL.defaultAccessControlVcMatcher,
+          [
+            [rdf.type, [ACP.Matcher]],
+            [ACP.vc, [VC_ACCESS_GRANT]],
+          ],
+        ],
+      ]).graphs.default
```

Comment:
This allows read on the resource controlled by `DEFAULT_ACCESS_CONTROL_RESOURCE_URL` and its decendents.